Countries/DetailsViewController.swift

```diff
@@ -213,15 +215,15 @@ class DetailsViewController: UIViewController {
             imageView.contentMode = .scaleAspectFit
             
             if i % 2 == 0 {
-                let image = UIImage(named: "ellipse1")
+                let image = UIImage(named: Constants.redBallImageName)
                 imageView.image = image
             } else {
-                let image = UIImage(named: "ellipse2")
+                let image = UIImage(named: Constants.greenBallImageName)
```

Comment:
both UIImage and image name are constants. please do like this: static let redBallImage = UIImage(named: "ellipse1")